test/pages/Pages.js

```diff
@@ -1,11 +1,18 @@
 const { InventoryPage } = require('./Inventory.page');
 const { LoginPage } = require('./Login.page');
 const { ShopingCartPage } = require('./ShopingCart.page');
+const {  CheckoutYourInformation } = require('../pages/Checkout.page.js');
+const {  CheckoutOverview } = require('../pages/CheckoutOverview.page.js');
 
 module.exports = {
     pages: {
         loginPage: new LoginPage(),
         inventoryPage: new InventoryPage(),
         shopingCartPage: new ShopingCartPage(),
+        CheckoutYourInformation: new CheckoutYourInformation(),
```

Comment:
it's better to follow same approach - having all the page name starting form the samll letter
```suggestion
        checkoutYourInformation: new CheckoutYourInformation(),
```